bailey/bailey.py

```diff
@@ -94,17 +94,20 @@
 grammar = Grammar(
     r"""
     expr       = (entry / emptyline )*
-    entry      = hash headword comma pos ws senses subentry emptyline
+    entry      = hash headword comma pos ws senses subentry period emptyline
+    # entry      = hash headphrase comma pos ws senses subentry period emptyline
     hash       = (~"#")*
+    # headphrase = headword (ws headword)*
     headword   = ~"[A-Z 0-9 -]*"i
-    pos        = (ws ~"[a-z]+\.")+
+    pos        = (ws ~"[a-z]+[\., ]")+
     subentry   = (semicolon ws senses)*
     senses     = (sense comma)* sense
     sense      = (ml ws ml)* ml
     ml         = ~"[\u0d00-\u0d7f]*"
-    semicolon  = ~";"
+    semicolon  = ~"[;:]"
```

Comment:
in many places the keyboardists made typos where they put a : in the place of ;. Since we are not preserving the data, I thought of bypassing them.